packages/react-data-grid/src/ChildRowDeleteButton.tsx

```diff
@@ -1,29 +1,20 @@
 import React from 'react';
-import classNames from 'classnames';
 
 export interface ChildRowDeleteButtonProps {
   treeDepth: number;
   cellHeight: number;
-  siblingIndex: number;
-  numberSiblings: number;
   onDeleteSubRow(): void;
   isDeleteSubRowEnabled: boolean;
-  allowAddChildRow?: boolean;
```

Comment:
This prop was never set.
It defaults to true, so only one of those classes was ever used, so I could clean up some css as well.
Then `siblingIndex` and `numberSiblings` were unused, so I could clean those up as well.

We don't use `subRowDetails.{siblingIndex,numberSiblings,canExpand}` internally, but we might externally, so I didn't clean those.